hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/test/java/org/apache/hadoop/yarn/client/api/impl/TestTimelineClient.java

```diff
@@ -89,7 +89,7 @@ public void tearDown() throws Exception {
     if (isSSLConfigured()) {
       KeyStoreTestUtil.cleanupSSLConfig(keystoresDir, sslConfDir);
     }
-    TimelineConnector.DEFAULT_SOCKET_TIMEOUT = 60_000;
+    client.getConnector().setSocketTimeOut(60_1000);
```

Comment:
Hi @slfan1989 !

Thanks for the patch!
If i see well we have a typo here, 60_1000 seems too much.

Reply:
@K0K0V0K Thanks for the review! I will improve it.